core-backend/common/src/runtime.rs

```diff
@@ -20,9 +20,25 @@
 
 use crate::{
     memory::{MemoryAccessRecorder, MemoryOwner},
-    BackendExternalities, BackendState, TerminationReason,
+    BackendExternalities, BackendExternalitiesError, BackendState, TerminationReason,
 };
 use gear_core::{costs::RuntimeCosts, pages::WasmPage};
+use gear_core_errors::ExtError as FallibleExtError;
+
+#[derive(Debug, Clone, derive_more::From)]
+pub enum RunFallibleError {
```

Comment:
I understand why `Run*` is chosen, but the sense is "run fallible syscall". Something related to that must be stated. Maybe it requires changing names in `Runtime` trait

Reply:
to be considered